tests/conftest.py

```diff
@@ -55,13 +54,7 @@ class Course(Base):
         name = sa.Column(sa.String(255), nullable=False)
         # These are for better model form testing
         cost = sa.Column(sa.Numeric(5, 2), nullable=False)
-        description = sa.Column(
-            sa.Text,
-            nullable=True,
-            info=dict(
-                marshmallow=dict(validate=[validate.Length(max=1000)], required=True)
-            ),
-        )
+        description = sa.Column(sa.Text, nullable=True)
```

Comment:
Passing `info=dict(marshmallow=...)` is depreciated so I remove from here and I add a column to test the behavior in the specific test. This stops warnings being generated for other tests that aren't testing that particular behavior.